requirements-testing.txt

```diff
@@ -3,5 +3,5 @@ pytest-cov
 ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow NWBFile load at all timess
 parameterized>=0.8.1
 ndx-miniscope
-spikeinterface[qualitymetrics]>=0.100.0
+spikeinterface[qualitymetrics] @ git+https://github.com/SpikeInterface/spikeinterface.git@main
```

Comment:
So you're proposing we merge with only the testing requirements pinned to latest SI dev?

Reply:
I am not sure why I did this or whether it is necessary. Removing it.